browser/main/lib/dataApi/reorderFolder.js

```diff
@@ -0,0 +1,46 @@
+const _ = require('lodash')
+_.move = require('lodash-move').default
+const path = require('path')
+const resolveStorageData = require('./resolveStorageData')
+const CSON = require('@rokt33r/season')
+
+/**
+ * @param {String} storageKey
+ * @param {number} oldIndex
+ * @param {number} newIndex
+ *
+ * @return {Object}
+ * ```
+ * {
+ *   storage: Object
+ * }
+ * ```
+ */
+function reorderFolder (storageKey, oldIndex, newIndex) {
+  let rawStorages
+  let targetStorage
+  try {
+    if (!_.isNumber(oldIndex)) throw new Error('oldIndex must be a number.')
+    if (!_.isNumber(newIndex)) throw new Error('newIndex must be a number.')
+
+    rawStorages = JSON.parse(localStorage.getItem('storages'))
+    if (!_.isArray(rawStorages)) throw new Error('Target storage doesn\'t exist.')
+
+    targetStorage = _.find(rawStorages, {key: storageKey})
+    if (targetStorage == null) throw new Error('Target storage doesn\'t exist.')
```

Comment:
Can you use `findStorage()`? https://github.com/BoostIO/Boostnote/blob/master/browser/lib/findStorage.js

Reply:
Why is `findStorage` in `browser/lib` and not in `browser/main/lib/dataApi`?
Besides my `reorderFolder` I also updated `updateFolder`.
There was one more possible candidate `renameStorage`, but I didn't want to touch it now in the scope of this issue.

Reply:
Because it just operates `localStorage`, does not operate files.